packages/error-reporting/.travis.yml

```diff
@@ -0,0 +1,8 @@
+sudo: false
```

Comment:
Will Travis work like this, only running within a directory in a larger project? We don't use Travis anymore since we switched to CircleCI. Is it possible that all of our merges to master will run the error-reporting test suite?